### PR TITLE
start styling

### PR DESCRIPTION
**Pull request by bmorelli25**
Starting to figure out props and find the general layout of the board.

### CODE REVIEW

File: src/components/Player.js [hunk 1]
```diff
@@ -2,12 +2,14 @@ import React from "react";
...
-import { StyledPlayer } from "../styles/StyledPlayer";
+import { StyledPlayer, StyledPlayerName } from "../styles/StyledPlayer";
 
 function Player({ name, cards }) {
   return (
     <StyledPlayer>
-      {name}
+      <StyledPlayerName>
+        <div>{name}</div>
+      </StyledPlayerName>
```
Comment (bmorelli25) [inline src/components/Player.js]: What's the reason for this inner div? Can we get rid of it to clean things up? Looks the same on my comp when I do.

Comment (bmorelli25) [reply src/components/Player.js]:
```suggestion
      <StyledPlayerName>{name}</StyledPlayerName>
```

Comment (dstrizzi) [reply src/components/Player.js]: This was to give margin after the names. The player cards weren't aligned on mine without it

Comment (bmorelli25) [reply src/components/Player.js]: Interesting. Lemme play around with it then. Now that score is on that side, maybe we need another styled component

Comment (dstrizzi) [reply src/components/Player.js]: ooooh, I see what you mean. ok yea

**Review by dstrizzi — APPROVED** [review]
Good to go

**Pull request merged**